app/models/patient.rb

```diff
@@ -906,7 +907,8 @@ def self.from_fhir(patient)
       preferred_contact_time: PatientHelper.from_preferred_contact_time_extension(patient),
       symptom_onset: PatientHelper.from_symptom_onset_date_extension(patient),
       last_date_of_exposure: PatientHelper.from_last_exposure_date_extension(patient),
-      isolation: PatientHelper.from_isolation_extension(patient)
+      isolation: PatientHelper.from_isolation_extension(patient),
+      jurisdiction: PatientHelper.from_full_assigned_jurisdiction_path_extension(patient, default_jurisdiction)
```

Comment:
So in testing locally I noticed that a history item isn't created on the record when changing jurisdiction via the API. That's because the actual attribute on the `Patient` is the `jurisdiction_id`, which is what `patient_diff()` in `patient.rb` looks for. Could we update this to grab the ID so it's more accurate with the `Patient` attributes and so that the history item shows up?

Reply:
Good catch. Should be good now with https://github.com/SaraAlert/SaraAlert/pull/494/commits/271a98631e5f60312f68068382481e331797f6bd